assets/src/edit-story/elements/text/edit.js

```diff
@@ -161,6 +165,19 @@ function TextEdit({
     [editorState, offset]
   );
 
+  // Handle basic key commands such as bold, italic and underscore.
+  const handleKeyCommand = getHandleKeyCommand(updateEditorState);
+
+  // Make sure to allow the user to click in the text box while working on the text.
+  const onClick = (evt) => {
+    const editor = editorRef.current;
+    // Refocus the editor if the container outside it is clicked.
+    if (!editor.editorContainer.contains(evt.target)) {
+      editor.focus();
+    }
+    evt.stopPropagation();
+  };
+
```

Comment:
What's with the reordering? Just curious.

Reply:
Oh. I wanted the `handleKeyCommand` to be closer to the `updateEditorState` since they are related. So I moved effects down.